<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>400</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string notr="true">Vatsinator</string>
  </property>
  <property name="windowIcon">
   <iconset resource="imgs.qrc">
    <normaloff>:/mainIcon/icon.png</normaloff>:/mainIcon/icon.png</iconset>
  </property>
  <widget class="QWidget" name="MainWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="margin">
     <number>1</number>
    </property>
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="5">
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="ReplaceableWidget" name="Replaceable" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>250</width>
          <height>13</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>250</width>
          <height>13</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="2" colspan="3">
       <widget class="QLabel" name="ClientsBox">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>13</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>13</height>
         </size>
        </property>
        <property name="text">
         <string>Clients:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="6">
       <widget class="QLabel" name="PositionBox">
        <property name="minimumSize">
         <size>
          <width>170</width>
          <height>13</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>170</width>
          <height>13</height>
         </size>
        </property>
        <property name="text">
         <string>N E</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="7">
       <widget class="MapWidget" name="MapDisplay" native="true">
        <property name="mouseTracking">
         <bool>true</bool>
        </property>
        <property name="focusPolicy">
         <enum>Qt::StrongFocus</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="MenuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="MenuVatsinator">
    <property name="geometry">
     <rect>
      <x>222</x>
      <y>165</y>
      <width>180</width>
      <height>234</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>150</width>
      <height>0</height>
     </size>
    </property>
    <property name="title">
     <string>Vatsinator</string>
    </property>
    <addaction name="ActionMetar"/>
    <addaction name="ActionRefresh"/>
    <addaction name="EnableAutoUpdatesAction"/>
    <addaction name="ActionFilterMode"/>
    <addaction name="separator"/>
    <addaction name="ActionFlightList"/>
    <addaction name="ActionATCList"/>
    <addaction name="separator"/>
    <addaction name="ActionExit"/>
   </widget>
   <widget class="QMenu" name="MenuHelp">
    <property name="minimumSize">
     <size>
      <width>150</width>
      <height>0</height>
     </size>
    </property>
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="ActionAbout"/>
   </widget>
   <widget class="QMenu" name="MenuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="ActionPreferences"/>
   </widget>
   <addaction name="MenuVatsinator"/>
   <addaction name="MenuSettings"/>
   <addaction name="MenuHelp"/>
  </widget>
  <action name="ActionExit">
   <property name="icon">
    <iconset theme="application-exit" resource="imgs.qrc">
     <normaloff>:/uiIcons/application-exit.png</normaloff>:/uiIcons/application-exit.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="ActionAbout">
   <property name="icon">
    <iconset resource="imgs.qrc">
     <normaloff>:/mainIcon/icon.png</normaloff>:/mainIcon/icon.png</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
   <property name="menuRole">
    <enum>QAction::AboutRole</enum>
   </property>
  </action>
  <action name="ActionMetar">
   <property name="icon">
    <iconset theme="weather-few-clouds" resource="imgs.qrc">
     <normaloff>:/uiIcons/weather-clouds.png</normaloff>:/uiIcons/weather-clouds.png</iconset>
   </property>
   <property name="text">
    <string>METARs...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+M</string>
   </property>
  </action>
  <action name="ActionRefresh">
   <property name="icon">
    <iconset theme="view-refresh" resource="imgs.qrc">
     <normaloff>:/uiIcons/view-refresh.png</normaloff>:/uiIcons/view-refresh.png</iconset>
   </property>
   <property name="text">
    <string>Refresh</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="ActionPreferences">
   <property name="icon">
    <iconset theme="preferences-system" resource="imgs.qrc">
     <normaloff>:/uiIcons/configure.png</normaloff>:/uiIcons/configure.png</iconset>
   </property>
   <property name="text">
    <string>Preferences: Vatsinator</string>
   </property>
  </action>
  <action name="ActionFlightList">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Flight list...</string>
   </property>
  </action>
  <action name="ActionATCList">
   <property name="text">
    <string>ATC list...</string>
   </property>
  </action>
  <action name="EnableAutoUpdatesAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Enable auto-updates</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="ActionFilterMode">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Filter mode</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MapWidget</class>
   <extends>QWidget</extends>
   <header>ui/widgets/mapwidget.h</header>
  </customwidget>
  <customwidget>
   <class>ReplaceableWidget</class>
   <extends>QWidget</extends>
   <header>ui/widgets/replaceablewidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="imgs.qrc"/>
 </resources>
 <connections/>
</ui>
